### ISSUE TITLE
Incompatible with latest Superpowered v1.4

### ISSUE THREAD

**Issue by IvanCaceres** [x86kernel/react-native-superpowered#2]
I'm using this library to start a new Audio instance, however experience an immediate crash upon running init code like:
`const music = new Audio('coolsong.mp3', 44100);`

I'm not seeing any stacktrace in Metro Bundler but Debugging / Running via Xcode provides a dump for Thread 1 and limited debugger output with no exception/error that can shed more light. After the crash the only thing in the debugger output is 'lldb'.

<img width="1792" alt="Xcode-Screenshot-CraftMusicCrash" src="https://user-images.githubusercontent.com/3813655/61349505-605c3e80-a832-11e9-8d83-3b24ec4533bd.png">

[Thread1CrashDump_CraftMusic.txt](https://github.com/x86kernel/react-native-superpowered/files/3400302/Thread1CrashDump_CraftMusic.txt)
[DebuggerOutput_CraftMusic_crash.txt](https://github.com/x86kernel/react-native-superpowered/files/3400304/DebuggerOutput_CraftMusic_crash.txt)

Currently a React Native/Superpowered crash is being triggered with initializing a new Audio instance, unfortunately this is something that was working before and now produces a crash every time, instantiating a new audio instance with (filepath, sampleRate)

I have tried getting around this in my existing project, and also tried a completley fresh react native project with ignite cli and i'm still receiving the same undeterminable crash when running new Audio();

my process for linking the native library has been to run react-native link

as well as adding the MediaPlayer framework via Xcode

Hoping you can shed light on the debug output which is seemingly obfuscated / obscured out. After a crash occurs the debugger output simply says: lldb

printing out arg1 (po $arg1): 
```
(lldb) po $arg1

CraftMusic was compiled with optimization - stepping may behave oddly; variables may not be available.

error: Couldn't materialize: couldn't read the value of register rdi

error: errored out in DoExecute, couldn't PrepareToExecuteJITExpression
```

register read: 
```
(lldb) register read

General Purpose Registers:

       rbx = 0x0000000000000002

       rbp = 0x00007ffeef7a1670

       rsp = 0x00007ffeef7a1630

       r12 = 0x0000000100776580  CraftMusic`::__24+[Audio createInstance:]_block_invoke() at Audio.mm:23

       r13 = 0x00006000009a8240

       r14 = 0x0000600002e4e700

       r15 = 0x0000000000000000

       rip = 0x00000001007826a7  CraftMusic`SuperpoweredAdvancedAudioPlayer::SuperpoweredAdvancedAudioPlayer(void*, void (*)(void*, SuperpoweredAdvancedAudioPlayerEvent, void*), unsigned int, unsigned char, unsigned int, unsigned int) + 433

13 registers were unavailable.
```

**Comment by IvanCaceres** [comment]
Solved: Superpowered has recently updated to version 1.4 and this project is currently incompatible.

Using Superpowered version 1.3.3 has allowed this bridge to work again.

**Comment by x86kernel** [comment]
@IvanCaceres Nice! i will temporarily fix superpowered sdk version to 1.3.3 cause I'm a little busy these days.
I'll upgrade the version later when I have time.
Thanks for your issue!
